posthog/api/person.py

```diff
@@ -231,7 +241,12 @@ class PersonViewSet(TeamAndOrgViewSetMixin, viewsets.ModelViewSet):
     queryset = Person.objects.all()
     serializer_class = PersonSerializer
     pagination_class = PersonLimitOffsetPagination
-    throttle_classes = [ClickHouseBurstRateThrottle, PersonsThrottle]
+    throttle_classes = [
```

Comment:
That's very nice, will make it easy to create some kind of dynamic throttle - will try writing one when Im back